waku/waku_store_sync/common.nim

```diff
@@ -0,0 +1,80 @@
+{.push raises: [].}
+
+import std/[options], chronos, stew/[byteutils]
+
+import ../waku_core
+
+const
+  DefaultSyncInterval*: Duration = 5.minutes
+  DefaultSyncRange*: Duration = 1.hours
+  RetryDelay*: Duration = 30.seconds
+  SyncReconciliationCodec* = "/vac/waku/reconciliation/1.0"
+  SyncTransferCodec* = "/vac/waku/transfer/1.0"
+  DefaultGossipSubJitter*: Duration = 20.seconds
+
+type
+  Fingerprint* = array[32, byte]
+
+  ID* = object
+    time*: Timestamp
+    fingerprint*: Fingerprint
```

Comment:
I'm thinking of changing this as `fingerprint` should be used only for incremental not msg hashes.